### ISSUE TITLE
Message not received, no icon

### ISSUE THREAD

**Issue by feklee** [mozillahispano/openwapp#108]
Last night I sent a message, and the recipient did not receive it. She, however, received the messages before and after that. Curiously, associated with the message is no icon. Seeing that there is no icon, I restarted the phone, and reconnected with OpenWapp, making sure it is online, but to no avail: The icon was still missing.

![Edited screenshot, showing no icon for message](https://cloud.githubusercontent.com/assets/14994/6847586/9d79ddbe-d3bf-11e4-9726-4dcf630515b5.png)

rev: 1.7.4-0-ge417a1a
OS: Boot2Gecko 1.3.0.0-prerelease
Device: Geeksphone Keon

**Comment by bobcanthelpyou** [comment]
The "no icon" problem is a duplicate of #11

**Comment by Timmaks** [comment]
Can you check if this is still a issue in the current version?

**Comment by feklee** [comment]
Hard to tell because the issue appeared randomly, maybe only once, and I don't use WhatsApp anymore.

**Comment by Timmaks** [comment]
ok thanks, closing. Please reopen when anyone sees this issue again
